helpers.cpp

```diff
@@ -194,3 +195,23 @@ void sepia(const int &height, const int &width, RGBTRIPLE **image) {
         }
     }
 }
+
+// This Function checks file format
+bool checkFile(char * file) {
+    
+    std::string storeFormat {};
+
+    for (int i{0}; file[i] != '\0'; i++)
+    {
+        if (file[i] == '.') {
+            for (int j{i}; file[j] != '\0'; j++)
+                storeFormat += file[j];
+            break;
+        }            
+    }
+
+    if (storeFormat == ".bmp")
+        return true;
+    
+    return false;
+}
```

Comment:
```suggestion
// Extracts file extension and checks if file is bmp
bool checkFile(char *file) {
    std::string storeFormat {};

    for (int i {0}; file[i] != '\0'; ++i) {
        if (file[i] == '.') {
            for (int j {i + 1}; file[j] != '\0'; ++j) {
                storeFormat += file[j];
            }
            
            break;
        }            
    }

    if (storeFormat == "bmp") {
        return true;
    }
    
    return false;
}
```

Reply:
Just added `i + 1` so that it will not start from `.` and start after `.` so that it can store only the name of the extension of image file.